tests/stevens.spec.ts

```diff
@@ -17,7 +17,7 @@ test('can find product using ASIN', async ({ page }) => {
   const asinText = await asin.textContent() as string;
   await searchForProduct(page, asinText);
 
-  const productSearchResult = page.getByText('        Enjoi Rasta Panda R7 Skateboard Deck       ', { exact: true });
+  const productSearchResult = page.getByText('        Powell Peralta Geegah Ripper Skate Deck       ', { exact: true });
   await expect(productSearchResult).toBeVisible();
 });
```

Comment:
I'd probably just delete this file.

Also, I noticed if you choose `fullyParallel: false` in the Playwright config, all but 1 of your tests passes.

This one still fails:
```bash
  1) [chrome] › addingtocart.spec.ts:18:9 › Adding to Cart › can add product to cart and deny warranty

    Error: locator.click: Element is outside of the viewport
    Call log:
      - waiting for getByTestId('attachSiNoCoverage-announce')
        - locator resolved to <span aria-hidden="true" class="a-button-text" id="attachSiNoCoverage-announce"> No Thanks </span>
      - attempting click action
        - scrolling into view if needed
        - done scrolling


      29 |             await expect(noThanksButton).toBeVisible()
      30 |
    > 31 |             await noThanksButton.click({ force: true })
         |                                  ^
      32 |
      33 |         }
      34 |         else {
        at /Users/stevenboutcher/Desktop/SocialQAproject/tests/addingtocart.spec.ts:31:34
```
Attaching the image from the trace at the point where the test failed:
<img width="1439" alt="Screenshot 2024-12-21 at 10 04 33 AM" src="https://github.com/user-attachments/assets/f7df199f-66ed-43a0-bd6e-0855d5d8c534" />

Reply:
I found the root cause of the failures: the warranty panel is giving a false indication that it is ready for interaction.

If you wait 500ms before checking if the warranty panel is visible, you'll get a more accurate reading:
```javascript
        //if the warranty panel is visible, click the button to say no thanks to the warranty
        const warrantyPanel = page.getByTestId('attach-desktop-sideSheet')
        await page.waitForTimeout(500);
        const warrantyPanelVisible = await warrantyPanel.isVisible()
```

Obviously, it's better to "intelligently wait" for an element that is only accessible if the warranty panel is visible. I'd experiment with expecting the "No thanks" button to be visible instead of the full warranty panel, for example.

<img width="819" alt="Screenshot 2024-12-21 at 10 30 16 AM" src="https://github.com/user-attachments/assets/97b55082-dda8-4f1c-a7c2-624381b554c3" />

Reply:
- ✅  Deleted ```steven.spec.ts``` as well as it's correlating ```utils.ts``` file

- ✅ changed ```fullyParallel: false``` but could you explain the affect of toggle this? Specifically in regards to this scenario?   
- ✅ changed tests to ```expect`` the "no thanks" and "warranty checkboxes" to be visible instead of the full warranty panel itself. Worked like a charm! Thanks! 
- ✅ Used auto format in VScode on all files to improve readability